### PR TITLE
iOS: Implement Swift rules part 2

### PR DESCRIPTION
**Pull request by karolpiateknet**
### Implement Swift rules part 2
```
- Add sqlite3_finalize case to sqlite3
- Add Realm database rule to Swift and Objc
- Add Certificate pinning rule
- Add Coredata rule
- Add Realm encryption rule
- Add Copy feature disabled rule
- Add Opening App with URL rule
```

### Checklist for PR

- [x] Run MobSF unit tests and lint `tox -e lint,test`.
- [x] Tested Working on Linux, Mac, Windows, and Docker
- [x] Make sure build is passing on your PR. [Build Status](https://travis-ci.com/MobSF/Mobile-Security-Framework-MobSF/pull_requests)

### CODE REVIEW

File: StaticAnalyzer/views/ios/common_rules.py [hunk 1]
```diff
@@ -77,13 +77,14 @@
...
+        'string2': 'sqlite3_finalize',
         'level': 'info',
-        'match': 'single_string',
+        'match': 'string_or',
         'input_case': 'exact',
         'cvss': 0,
         'cwe': '',
         'owasp': '',
-        'owasp-mstg': OWASP_MSTG['storage-2'],
+        'owasp-mstg': OWASP_MSTG['storage-14'],
```
Comment (ajinabraham) [inline StaticAnalyzer/views/ios/common_rules.py]: Why not `storage-14` ? Isn't this similar to `[realm transactionWithBlock:` check?

File: StaticAnalyzer/views/ios/swift_rules.py [hunk 2]
```diff
@@ -262,4 +262,82 @@
         'owasp': '',
         'owasp-mstg': OWASP_MSTG['storage-5'],
     },
+    {
+        'desc': 'This App may have Certificate Pinning mechanizm.',
```
Comment (ajinabraham) [inline StaticAnalyzer/views/ios/swift_rules.py]: Typo `mechanism`

File: StaticAnalyzer/views/ios/swift_rules.py [hunk 3]
```diff
@@ -262,4 +262,82 @@
...
+    {
+        'desc': 'Opening App with URLs can lead to potencial security leaks.',
+        'type': 'regex',
+        'regex1': r'func application\(.*open url: URL',
+        'level': 'warning',
+        'match': 'single_regex',
+        'input_case': 'exact',
+        'cvss': 0.0,
+        'cwe': CWE['CWE-939'],
+        'owasp': '',
```
Comment (ajinabraham) [inline StaticAnalyzer/views/ios/swift_rules.py]: M1: Improper Platform Usage ?

File: StaticAnalyzer/views/ios/common_rules.py (hunk 1 above)
Comment (karolpiateknet) [reply StaticAnalyzer/views/ios/common_rules.py]: Missed that, I changed storage-2 to storage-14 in `[realm transactionWithBlock:` rule

**Pull request merged**